src/Container/Context.php

```diff
@@ -32,41 +32,43 @@ public function currentDependency(): ?Dependency
     public function getName(): string
     {
         return match($this->reflector::class) {
-            ReflectionClass::class, ReflectionFunction::class => $this->reflector->getName(),
+            ReflectionClass::class => $this->reflector->getName(),
```

Comment:
I ran into some bugs with context naming when working on this PR. I fixed them here, but they aren't really related to this PR. There's room for further improvement and I made an issue for that here: https://github.com/tempestphp/tempest-framework/issues/185